e2e/config.ts

```diff
@@ -36,6 +36,9 @@ export class Config {
   public static readonly manageCasesBaseURL: string =
     process.env.MANAGE_CASES_BASE_URL ||
     "https://manage-case.aat.platform.hmcts.net/cases";
+  public static readonly caseWorkerManageCasesBaseURL: string =
+    process.env.MANAGE_CASES_BASE_URL ||
```

Comment:
If process.env.MANAGE_CASES_BASE_URL  is not null, would the URL be ../cases and not ../work?

Reply:
That doesn't need to be there - forgot to delete it!